### ISSUE TITLE
model bug

### ISSUE THREAD

**Issue by RYiHui** [joh-fischer/MCNN#2]
My code reports a error as follows,
![1666080152714](https://user-images.githubusercontent.com/90946052/196388351-4826a3ac-b7ba-496f-be58-76c63142ef66.png)

**Comment by RYiHui** [comment]
it seems torch.cat dim should be setted as 2, but it will occur another error.

**Comment by joh-schb** [comment]
Can you give me some information to reproduce the error? What is your input size? (Sequence length & dim)

**Comment by RYiHui** [comment]
thank you ,I have solve this problem,my input Sequence length and dim is 140 and 1 respectivly.(ECG5000) ,My edit as  follows:
![1666163604866](https://user-images.githubusercontent.com/90946052/196621929-c096ef23-a4c9-4d47-9b54-f9825ac7b0b5.png)
